code/modules/cargo/expressconsole.dm

```diff
@@ -64,7 +64,7 @@
 			) // see, my quartermaster taught me a few things too
 		if((P.hidden) || (P.special)) // like, how not to rip the manifest
 			continue// by using someone else's crate
-		if(!(obj_flags & EMAGGED) && P.contraband) // will you show me?
+		if(!(obj_flags & EMAGGED || contraband) && P.contraband) // will you show me?
```

Comment:
This is extremely risky and will likely lead to erronous behaviour. All bit operations should be wrapped in parenthesis due to their order of operation not being what you would expect.

Reply:
Either
`if(!((obj_flags & EMAGGED) || contraband) && P.contraband)`
Or
`if(!(obj_flags & EMAGGED) && !contraband && P.contraband)`